package.json

```diff
@@ -13,7 +13,15 @@
     "lint-fix": "eslint --fix --ext .ts .",
     "validate": "yarn audit --level moderate --groups dependencies --recursive",
     "clean": "rimraf dist",
-    "reset": "rimraf --glob dist node_modules coverage *.log"
+    "reset": "rimraf --glob dist node_modules coverage *.log",
+    "start-database": "ts-node-script ./database/manager/mongo.ts --start",
```

Comment:
Should not be of a difference see https://github.com/TypeStrong/ts-node/blob/main/package.json#L41 , https://github.com/TypeStrong/ts-node/blob/master/src/bin-script.ts aiming portability since it locates a tsconfig relative to the location of the script, not to cwd.

Reply:
If there is no difference, then I prefer just having `ts-node`. I have been unable to find documentation on the `--script-mode flag`.